manifests/params.pp

```diff
@@ -6,14 +6,17 @@
     'transmission-daemon',
   ]
 
-  if $facts['os']['release']['full'] == '16.04' {
+  #if $facts['os']['release']['full'] == '16.04' {
+  if ($::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '16.04')) {
```

Comment:
This is the old Puppet3 way of using facts. Please update this to use the new `$facts` hash.